src/BenchmarkDotNet/ConsoleArguments/ConfigParser.cs

```diff
@@ -425,7 +417,7 @@ private static Job MakeWasmJob(Job baseJob, CommandLineOptions options, TimeSpan
             bool wasmAot = options.AOTCompilerMode == MonoAotCompilerMode.wasm;
 
             var wasmRuntime = new WasmRuntime(
-                mainJs: options.WasmMainJs ?? new FileInfo(Path.Combine(options.RuntimeSrcDir.FullName, "src", "mono", "wasm", "runtime-test.js")),
+                mainJs: new FileInfo(Path.Combine(options.RuntimeSrcDir.FullName, "src", "mono", "wasm", "runtime-test.js")),

```

Comment:
is it being used anywhere now? It seems to be hardcoded in the project file:

```xml
<WasmMainJSPath>$(RuntimeSrcDir)\src\mono\wasm\runtime-test.js</WasmMainJSPath>
```

Reply:
I removed it.